redfish-core/lib/log_services.hpp

```diff
@@ -2675,14 +2650,40 @@ inline void requestRoutesDBusEventLogEntry(App& app)
             return;
         }
 
-        auto updatePropertyCallback =
-            [&req, asyncResp, entryId](bool hiddenPropVal) {
+        std::optional<bool> resolved;
```

Comment:
I guess I could have put this code into a function and just called it from the EventLog and CELog patch functions.

Reply:
Yeah, that would be cleaner, up to you

Reply:
I think I'll just leave it how it is.